README.md

```diff
@@ -2,372 +2,10 @@
-IMAGE NAME                        TAG                       DIGEST    SIZE
-c3/openjdk-dev                    commit-2674e8a-squashfs   b545b8ba  321MB
-c3/openjdk-dev                    commit-d5024ec-squashfs   cd45f8cf  321MB
-```
-
-# Sync (pull-based mirroring)
-Periodically pull and synchronize images between zot registries.
-The synchronization is achieved by copying all the images found at source to destination.
-To use it see [sync-config](examples/config-sync.json)
-Supports:
-  - TLS verification
-  - Prefix filtering (can contain multiple repos, eg repo1/repoX/repoZ)
-  - Tags regex filtering
-  - Tags semver compliance filtering (the 'v' prefix is optional)
-  - BASIC auth
-  - Image signatures
-
-# Benchmarking
-
-You can benchmark a zot registry or any other dist-spec conformant registry with `zb`.
-
-## Building `zb`
-
-```console
-$ make bench
-```
-
-will produce `bin/zb` binary.
-
-## Running `zb`
-
-See [details](./cmd/zb/README.md)
-
-# Ecosystem
-
-## skopeo
-
-[skopeo](https://github.com/containers/skopeo) is a tool to work with remote
-image repositories.
-
-* Pull Images
-
-```
-skopeo copy docker://<zot-server:port>/repo:tag docker://<another-server:port>/repo:tag
-```
-
-* Push Images
-
-```
-skopeo copy --format=oci docker://<another-server:port>/repo:tag docker://<zot-server:port>/repo:tag
-```
-
-## cri-o
-
-[cri-o](https://github.com/cri-o/cri-o) is a OCI-based Kubernetes container
-runtime interface.
-
-Works with "docker://" transport which is the default.
-
-# Metrics
-
-Can be used for both dist-spec-only zot & the zot with all extensions enabled
-
-## Node Exporter
-The dist-spec-only zot exposes internal metrics into a Prometheus format through a node exporter.
-The configuration of node exporter contains connection details for the zot server it is intend to scrape metrics from. See a [configuration example](./examples/metrics/exporter/config-minimal.json). The metrics are automatically enabled in the zot server on first scrape from the Node Exporter (no extra configuration option is needed). Similarly, the metrics are automatically disabled when Node Exporter did not perform any scrapings in a while.
-
-```
-bin/zxp config _config-file_
-```
-
-## Enable Metrics
-In the zot with all extensions case see [configuration example](./examples/config-metrics.json) for enabling metrics
-
-## Image linting
-
-# Mandatory Annotations
-When pushing an image, if the mandatory annotations option is enabled, linter will verify if the mandatory annotations list present in the config is also found in the manifest's annotations list. If there are any missing annotations, the push will not take place.
-
-## Clustering
-
-zot supports clustering by using multiple stateless zot with shared s3 storage and a haproxy (with sticky session) in front of them.
-
-- haproxy [configuration example](./examples/cluster/haproxy.cfg)
-- zot s3 [configuration example](./examples/config-s3.json)
-
-# Contributing
```

Comment:
Why remove this?

Reply:
Everything lands in https://zotregsitry.io as a single source of truth